### ISSUE TITLE
Remove declaration of terms from external ontologies

### ISSUE THREAD

**Issue by rjyounes** [buda-base/owl-schema#181]
BDO should not include term definitions from external ontologies. This applies to the following namespaces (in prefixed form):

```
iiif2
iiif3
dct
foaf
owl
rdfs
skos
vcard
xsd
```

For example:

```
dct:modified
owl:Thing
xsd:date
```
etc.

Note that it some cases the BDO declaration is incorrect and thus in conflict with the original terms: `skos:closeMatch` is an object property rather than an annotation property; similarly for `owl:sameAs`. 

These should be removed. You cam import these ontologies, if you want to take advantage of inferences they provide, or simply use the terms without importing. 

Élie: I am willing to make this change if you agree to it.

**Comment by eroux** [comment]
Yes good idea! One issue though is the translations of the associated labels in Tibetan and Chinese... I'm not really sure what the best way would be to handle that, but an interim solution could be to move everything we have in these namespaces into a separate file... wdyt?

**Comment by rjyounes** [comment]
I don't see any cases where you are doing that. Were you planning to?

I don't think the separate file solves the problem, and it seems to me for external terms you should live with the fact that the ontology doesn't specify annotation values in other languages. If you're using a term from another ontology then the tacit agreement is that you are using it as is. I suppose you could define your own properties or classes that are subproperties or subclasses of the external ones, but this is not very palatable - it's a lot of extra baggage in your ontology, and your terms will not easily be recognized and interpreted by others. 

Thoughts?

**Comment by eroux** [comment]
[Here](https://github.com/buda-base/owl-schema/blob/master/translations/core_bo.ttl#L2922) are a few examples

Well, our web UI works by looking at the labels of the properties in the ontology and we need to display labels for these external terms so for our platform we'll keep these triples, but we can put them in a more discrete place like on another repository.

This would require to extract them in a separate file so that I can move them... could you do that as part of the change? (also, please do it in a pull request, so that I can make sure the new system works, it's all in production)

**Comment by eroux** [comment]
yes, let's skip everything but the labels, that should work

**Comment by eroux** [comment]
We are using transifex to do the translation and then we import the output in these files. They are handled by https://github.com/buda-base/owl-schema/blob/master/ont-policy.rdf#L209 which drives out software so these files get merged in our ontology. Ideally if you could extract all the labels of all the properties in the third party namespaces (in all the files, including the translations) into a separate `third_part_labels.ttl` that would be great!
